src/ssl.c

```diff
@@ -16832,6 +16832,26 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
 
         return 0;
     }
+
+    int wolfSSL_MD5_Transform(WOLFSSL_MD5_CTX* md5, const unsigned char* data)
+    {
+        int ret;
+
+       WOLFSSL_ENTER("MD5_Transform");
+       #if defined(BIG_ENDIAN_ORDER)
+       {
+            ByteReverseWords((word32*)data, (word32*)data, WC_MD5_BLOCK_SIZE);
+       }
+       #endif
+
+       ret = wc_Md5Transform((wc_Md5*)md5, data);
```

Comment:
It doesn't look like wc_Md5Transform() checks md5 or data against NULL.  We should check/sanitize those arguments coming into wolfSSL_MD5_Transform() first.

Reply:
Exactly. Thank you for pointing out!